src/components/admin/UserProfileModal/components/ProfileModalContent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { UserWithId } from "types/id";
+
+import { useWorldAndSpaceByParams } from "hooks/spaces/useWorldAndSpaceByParams";
+import { useProfileQuestions } from "hooks/useProfileQuestions";
+
+import { Badges } from "components/organisms/Badges";
+
+import { ProfileModalButtons } from "./buttons/ProfileModalButtons";
+import { ProfileModalBasicInfo } from "./header/ProfileModalBasicInfo";
+import { ProfileModalLinks } from "./links/ProfileModalLinks";
+import { ProfileModalQuestions } from "./ProfileModalQuestions";
+
+export interface ProfileModalContentProps {
+  user: UserWithId;
+  onPrimaryButtonClick: () => void;
+  onEditMode?: () => void;
+  onModalClose: () => void;
+}
+
+export const ProfileModalContent: React.FC<ProfileModalContentProps> = ({
+  user,
+  onPrimaryButtonClick,
+  onEditMode,
+  onModalClose,
+}) => {
+  const { worldId, space } = useWorldAndSpaceByParams();
+  const { questions, answers } = useProfileQuestions(user, worldId);
+
+  return (
+    <div className="ProfileModalContent">
```

Comment:
You can remove `ProfileModalContent` and maybe replace the `div` with `Fragment`

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8